code/game/machinery/computer/card.dm

```diff
@@ -467,7 +467,7 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 						to_chat(usr, "<span class='alert'>No log exists for this job.</span>")
 						updateUsrDialog()
 						return
-					if(inserted_modify_id.registered_account)
+					if(inserted_modify_id.registered_account && (alert(usr, "Would you like to set their bank account's assigned job to this?", "Bank Account Job", "Yes", "No") == "Yes"))
 						inserted_modify_id.registered_account.account_job = jobdatum // this is a terrible idea and people will grief but sure whatever
```

Comment:
Should be moved to a new proc with waitfor set to FALSE, otherwise that alert() will halt the current function, possibly causing some runtimes or issues if the ID were removed or changed in the meanwhile